nixos/tests/all-tests.nix

```diff
@@ -288,8 +288,8 @@ in {
   activation-etc-overlay-mutable = runTest ./activation/etc-overlay-mutable.nix;
   activation-etc-overlay-immutable = runTest ./activation/etc-overlay-immutable.nix;
   activation-perlless = runTest ./activation/perlless.nix;
-  etcd = handleTestOn ["x86_64-linux"] ./etcd.nix {};
-  etcd-cluster = handleTestOn ["x86_64-linux"] ./etcd-cluster.nix {};
+  etcd = handleTestOn [ "aarch64-linux" "x86_64-linux" ] ./etcd/etcd.nix {};
+  etcd-cluster = handleTestOn [ "aarch64-linux" "x86_64-linux" ] ./etcd/etcd-cluster.nix {};
```

Comment:
The etcd-cluster test seems pretty flaky for me (80 Core Ampere Altra) - at least on the first couple of runs - I'll take a second look later.

At first glance tho it looks like an IO issue on writes (I assume related to single core perf)

Reply:
@endocrimes Can you provide an update on this?

Reply:
OfBorg is green for `aarch64-linux`:
![image](https://github.com/NixOS/nixpkgs/assets/5861043/0b35ca8e-8436-4101-a3de-df5f0c9d8a91)